### PR TITLE
Add some more linting

### PR DESCRIPTION
**Pull request by alexfikl**
Commits should be fairly separate and self-contained.

(Ran the loopy CI locally and this didn't seem to break anything!)

### CODE REVIEW

**Comment by alexfikl** [comment]
Looks like the CI isn't set up to also install `pymbolic` and `pyopencl` and it's annoying `pylint`. Any preference on how to handle that?
* for `pymbolic`, it seems to use `gcd`, which is also in `math.gcd`. Would replacing it with that suffice?
* for `pyopencl`, it just needs a `dtype_to_ctype`. Just ignore it?

File: cgen/__init__.py [hunk 1]
```diff
@@ -457,9 +461,9 @@ def __init__(self, tpname, fields, declname=None,
         padded_bytes = ((bytes + align_bytes - 1) // align_bytes) * align_bytes
 
         def satisfies_primality(n):
-            from pymbolic.algorithm import gcd
+            import math
             for p in aligned_prime_to:
-                if gcd(n, p) != 1:
+                if math.gcd(n, p) != 1:
                     return False
             return True
```
Comment (alexfikl) [inline cgen/__init__.py:468]: Would this work?

Comment (inducer) [reply cgen/__init__.py:468]: Seems fine.

**Comment by inducer** [comment]
Could you add a loopy downstream?

**Comment by inducer** [comment]
>  Just ignore it?

Yep.

**Comment by alexfikl** [comment]
> Could you add a loopy downstream?

Sure thing!

**Comment by inducer** [comment]
LGTM, thanks!

**Pull request merged**